src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -27,18 +28,26 @@ import { selectCryptoMarketData, selectFiatToUsdRate } from '../marketDataSlice/
 
 const selectTradeQuoteSlice = (state: ReduxState) => state.tradeQuoteSlice
 
-// TODO(apotheosis): Cache based on quote ID
-export const selectSelectedQuote: Selector<ReduxState, TradeQuote2 | undefined> =
+export const selectSelectedSwapperName: Selector<ReduxState, SwapperName | undefined> =
+  createSelector(selectTradeQuoteSlice, swappers => swappers.swapperName)
+
+export const selectQuotes: Selector<ReduxState, ApiQuote[] | undefined> =
+  createDeepEqualOutputSelector(selectTradeQuoteSlice, swappers => swappers.quotes)
+
+export const selectSelectedQuoteApiResponse: Selector<ReduxState, ApiQuote | undefined> =
```

Comment:
now that there's 1 big api request for all swappers, we're selecting a quote rather than an api response

```suggestion
export const selectSelectedQuoteFromApiResponse: Selector<ReduxState, ApiQuote | undefined> =
```

Reply:
I'd be a fan of bringing back the wording `active` instead of `selected` purely because of redux `select` doubling up the words

Reply:
Fixed the name up in https://github.com/shapeshift/web/pull/4847/commits/c74339b12b9b75e9f5930866150318945d88049c, though agree, let's do another PR to revert the naming to active - too much selecting selected selectors.